includes/definitions/discovery/eatonpdu.yaml

```diff
@@ -10,6 +10,11 @@ modules:
                     multiplier: 1
                     descr: 'S{{ $index }} {{ $outletLabel }} Output'
                     index: 'outletActivePower.{{ $index }}'
+                -
+                    oid: inputTotalWatts
+                    num_oid: '.1.3.6.1.4.1.534.6.6.7.3.5.1.4.{{ $index }}'
+                    index: '0.1'
```

Comment:
the index definition should be dynamic (in case the inputTotalWatts is defined more than once). Please write it 
`index: 'inputTotalWatts.{{ $index }}' `